### ISSUE TITLE
[Feature Request] Completely ignore notifications for selected chats

### ISSUE THREAD

**Issue by Zeeex** [telegramdesktop/tdesktop#24556]
### Is your feature request related to a problem?

There is no option to fully prevent some chats from making notification badges in taskbar and tray icon. (Without disabling global notification badges). 
** Also I need to have notification badges for some muted chats, so that option is not viable **

I'm on Win10

### Describe the solution you'd like

One of the suggestions below would suffice. (Sorted from most favorable)
1. A per chat option to disable notification badges,
2. Make that the archived chats don't make badges,
3. Maybe some kind of "void" folder for chat/s that we want completely ignored

### Describe alternatives you've considered

None of the alternatives is what I need, but here they are:
- Exiting that (group) chat,
- Disabling notification badges for every chat,
- Disabling notifications for muted chats only,

### Additional context

Thanks for the wonderful app BTW! 😊💙

**Comment by ffactory-ofcl** [comment]
I've also been annoyed by this. In the web client, the badge only counts non-archived chats, but the desktop client counts them too.
I'm in a few big channels that have hundreds of messages per day. I archived them, so I only see them when I want to. I normally use the badge to see if I received an important message, but as all those unimportant messages from archived chats flood in, the badge is rendered useless.

**Comment by daald** [comment]
+1 for "Make that the archived chats don't make badges,"

I have a **muted** chat in **archive** and its unread messages are shown in the badge and always take my **attention**

**Comment by Mishin870** [comment]
+1 for "Make that the archived chats don't make badges"

Please do this. It takes a lot of time to read muted chats. I want to see the number on the telegram icon only for important chats, but i don't want to enable real notifications for them

**Comment by paul1149** [comment]
Good idea. I have a folder for non-contact chats. I do this because TG does not have a "block DMs from non-contacts" options, which would reduce spam by 100% for me. At least I would like everything that goes there to die and not demand my attention to mark it read.

**Comment by Zeeex** [comment]
It has been a year, yet we observe no signs of progress. 😢 

@john-preston 
Apologies for reaching out in this manner, but could you provide any information regarding when or if this issue will be resolved?

**Comment by john-preston** [comment]
@Zeeex I didn't have any plans for such changes right now.

**Comment by destroy** [comment]
Why is it still open? Any alternative clients with this feature?

**Comment by VidKuo** [comment]
I cannot believe this issue is still not fixed after 3 years. This is the ONLY chat client that I've seen completely ignore archive/muted channels. It makes using Telegram Desktop a complete pain. Please fix.

**Comment by ogarcia** [comment]
From my point of view, I don't see the connection between archiving a chat and it being of zero priority (completely muted).

IMHO, there should be a global option that works exactly the same in all clients, which would be “channel without priority or muted channel,” which would update normally but would not have any kind of unread notification. Only when you open the channel would you be taken to the last read message and see the number of messages you have left to read.

With the huge number of groups that exist and that are constantly generating content, this feature is essential today.

And as I mentioned at the beginning, this has nothing to do with having an archived channel. For me, an archived channel is something that is no longer active and that you want to keep stored for whatever reason.

**Comment by daald** [comment]
Yes, for me, the counter is a tool which gazes attention. important. priority. must read. NOW!

An archived chat is archive because at some time I decided it is not important.
And a muted chat is muted because at some time I decided it is too noisy.

There is also the "All groups" counter. It is very helpful, it also only counts the number of groups and not how messages are inside there. But it is not on the program icon, it is inside the main window which is fine.

@ogarcia I don't think you will find anyone who needs a counter outside the main window which shows you how many new archived and muted messages are there.
Just another option would only blow up the complexity of the client, with no benefit.

For me, the counter is currently useless, it causes **Notification fatigue**. It shows "..8" on my system, I have no Idea what I should do which this information. When I open the main window, there's noting I have to read right now.
For me it's important to make this feature useful again 👍

**Comment by ogarcia** [comment]
No, perhaps I didn't explain myself well.

What I mean is that I want an option that simply mutes the channel. From then on, that channel remains as if no one ever "spoke" on it again. Only when I enter that channel will I see the number of messages I haven't read and scroll to the last thing I read (as it works right now). But in the channel list,  notifications, etc., there is no counter or anything. And for me, that's independent of whether or not I have an archived channel.

The implementation is really very straightforward; it is as simple as adding a new “muted channel” option and, if selected, not displaying any counters anywhere and maintaining the current operation when you open the channel. It is actually similar to what Matrix does with low-priority channels.

**Comment by daald** [comment]
@ogarcia I would call that "hide unread status"
I wonder how many people really need this. I don't
